iac-extensions/docker/src/main/java/org/sonar/iac/docker/tree/api/NoneInstruction.java

```diff
@@ -19,5 +19,5 @@
  */
 package org.sonar.iac.docker.tree.api;
 
-public interface ShellFormTree extends LiteralListTree {
+public interface NoneInstruction extends Instruction {
```

Comment:
Now that I see it, not sure if None should be an Instruction.
Anyway, not in the scope of this ticket, I will note it to check this later.

Reply:
I think that git wrongly detects the move of the file.